SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorModule.cpp

```diff
@@ -80,6 +84,32 @@ FString FSpatialGDKEditorModule::GetSpatialOSCloudDeploymentName() const
 	return GetDefault<USpatialGDKEditorSettings>()->DevelopmentDeploymentToConnect;
 }
 
+bool FSpatialGDKEditorModule::ShouldConnectServerToCloud() const
+{
+	return GetDefault<USpatialGDKEditorSettings>()->IsConnectServerToCloudEnabled();
+}
+
+bool FSpatialGDKEditorModule::TryStartLocalReceptionistProxyServer() const
+{
+	if (ShouldConnectToCloudDeployment() && ShouldConnectServerToCloud())
+	{
+		bool bSuccess = LocalReceptionistProxyServerManager->TryStartReceptionistProxyServer(GetDefault<USpatialGDKSettings>()->IsRunningInChina(), GetDefault<USpatialGDKEditorSettings>()->DevelopmentDeploymentToConnect, GetDefault<USpatialGDKEditorSettings>()->ListeningAddress, GetDefault<USpatialGDKEditorSettings>()->LocalReceptionistPort);
```

Comment:
Nit (just because we're calling `GetDefault` for it 3 times and the line gets a little long)
```suggestion
		const USpatialGDKEditorSettings* EditorSettings = GetDefault<USpatialGDKEditorSettings>();
		bool bSuccess = LocalReceptionistProxyServerManager->TryStartReceptionistProxyServer(GetDefault<USpatialGDKSettings>()->IsRunningInChina(), EditorSettings->DevelopmentDeploymentToConnect, EditorSettings->ListeningAddress, EditorSettings->LocalReceptionistPort);
```